hack/keepalived-docker/manage-keepalived.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+export assigned_ip=172.22.0.2/24
+
+export interface=ironicendpoint
```

Comment:
this needs to come from the configmap, as env variable

Reply:
Done